llvm/utils/TableGen/Common/CodeGenRegisters.cpp

```diff
@@ -2221,20 +2209,21 @@ void CodeGenRegBank::computeRegUnitSets() {
     if (RCRegUnits.empty())
       continue;
 
-    LLVM_DEBUG(dbgs() << "RC " << RC.getName() << " Units:\n";
-               for (auto U
-                    : RCRegUnits) printRegUnitName(U);
-               dbgs() << "\n  UnitSetIDs:");
+    LLVM_DEBUG({
+      dbgs() << "RC " << RC.getName() << " Units:\n";
+      for (auto U : RCRegUnits)
+        printRegUnitName(U);
```

Comment:
Use `formatv` or `llvm::interleaved` to eliminate loop? 
But if this stays as is, don't "always auto".

Reply:
moved to helper function.